store_test.go

```diff
@@ -45,16 +43,7 @@ func TestStore(t *testing.T) {
 		require.NoError(t, store.Close())
 	})
 
-	t.Run("master db", func(t *testing.T) {
-		db, err := sql.Open("ramsql", "TestStore-master-db")
-		require.NoError(t, err)
-		defer db.Close()
-
-		_, err = db.Exec("CREATE TABLE test (id INT);")
```

Comment:
I have removed the actual SQL queries because with the RPC API, we would be actually testing the mocks. There's no benefit of testing just for the sake of it. I have only kept the master/replica verification logic.